src/device-registry/models/Event.js

```diff
@@ -376,6 +413,21 @@ eventSchema.statics = {
       let projection = {
         _id: 0,
       };
+      let meta = {
+        total: { $arrayElemAt: ["$total.device", 0] },
+        skip: { $literal: skip },
+        limit: { $literal: limit },
+        page: {
```

Comment:
Adding pagination to ensure that one can now fetch data using page numbers and not just skipping/